### PR TITLE
Import config should persist nothing

### PR DESCRIPTION
**Pull request by gaurav274**
1. Removed singleton classes, `ConfigurationManager`, `SQLConfig`, and `CatalogManager`. They are now encapsulated within the `EVADatabase` class, which is owned by each server instance.
2. The Pythonic APIs no longer rely on server-client architecture. The current implementation allows the Pythonic APIs to run serverless. This design choice was made for stability, and we plan to reintroduce the server-client architecture. Please note that the command-line client still utilizes the server-client architecture.
3. For backward compatibility, we renamed `connect` to `connect_remote` for notebooks that use old server-client architecture.
4. The `eva.yml` file is no longer used. In the current implementation, configuration changes cannot be made post-launch. We have plans to introduce a `set` command to support configuration modifications after launch.

### CODE REVIEW

File: eva/executor/storage_executor.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from typing import Iterator
 
 from eva.catalog.catalog_type import TableType
+from eva.database import EVADB
```
Comment (jarulraj) [inline eva/executor/storage_executor.py:18]: We could use `EVADatabase` instead of `EVADB` to avoid confusion with the system name

**Comment by jarulraj** [comment]
Does this PR also subsume #791?

**Comment by gaurav274** [comment]
> Does this PR also subsume #791?

No

**Review by jiashenC — COMMENTED** [review]
I actually like the previous approach that modules like CatalogManager, ConfiguraitonManagaer stays global and singleton and only initialized when called. So, we also do not need to pass instance around. Any particular reason that we adopt a new design?

**Review by xzdandy — APPROVED** [review]

**Pull request merged**